docs/screens/login_screen.md

```diff
@@ -0,0 +1,29 @@
+# Login Screen
+
+<img src="../analytics_imgs/login/login_screen.png" width=20% height=20%>
+
```

Comment:
**question**: Why there is no `screen_opened` for this screen?

Reply:
I don't know but I added it :)